docs/contributing/Masterlist-Editing.html

```diff
@@ -37,26 +57,25 @@ <h3>Writing Style</h3>
 <p>The above points are illustrated in the following example.</p>
 
 <pre><code>name: 'Oscuro''s_Oblivion_Overhaul.esm'
+url: [ 'https://www.nexusmods.com/oblivion/mods/15256' ]
 req:
   - name: 'example.esp'
-    display: '[Example Mod](http://www.example.com)'
+    display: '[Example Mod](https://www.example.com)'
     condition: 'version("Oscuro''s_Oblivion_Overhaul.esm", "15.0", ==)'
 tag:
-  - Actors.Spells
-  - Graphics
-  - Invent
-  - Relations
-  - Scripts
-  - Stats
   - name: -Relations
     condition: 'file("Mart''s Monster Mod for OOO.esm") or file("FCOM_Convergence.esm")'
 msg:
   - type: say
-    content: 'Do not clean. "Dirty" edits are intentional and required for the mod to function.'
-    lang: eng
-url: [ 'http://oblivion.nexusmods.com/mods/15256' ]</code></pre>
+    content:
+      - lang: en
+        text: 'This is an "Example".'
+      - lang: bg
+        text: 'Това е "Образец".'
+      - lang: de
+        text: 'Dies ist ein "Beispiel".'</code></pre>
 
-<pre><code>{name: Oscuro's_Oblivion_Overhaul.esm, req: [{ name: example.esp, display: '[Example Mod](http://www.example.com)', condition: 'version("Oscuro''s_Oblivion_Overhaul.esm", "15.0", ==)' }], tag: [ Actors.Spells, Graphics, Invent, Relations, Scripts, Stats, { name: -Relations, condition: file("Mart's Monster Mod for OOO.esm") or file("FCOM_Convergence.esm") }], msg: [{ type: say, content: Do not clean. "Dirty" edits are intentional and required for the mod to function., lang: eng }], url: [ http://oblivion.nexusmods.com/mods/15256 ]}</code></pre>
+<pre><code>  - {name: Oscuro's_Oblivion_Overhaul.esm, url: [ https://www.nexusmods.com/oblivion/mods/15256 ], req: [{ name: example.esp, display: '[Example Mod](https://www.example.com)', condition: 'version("Oscuro''s_Oblivion_Overhaul.esm", "15.0", ==)' }], tag: [{ name: -Relations, condition: file("Mart''s Monster Mod for OOO.esm") or file("FCOM_Convergence.esm") }], msg: [{ type: say, content: [{ lang: en, text: This is an "Example".}, { lang: bg, text: Това е "Образец".}, {lang: de, text: Dies ist ein "Beispiel".}]}] }</code></pre>
```

Comment:
Why was the `  - ` added to the start of this code block? That's the list item syntax in YAML's block formatting, but the purpose of this code block is to show the equivalent flow formatting to the block formatting code block above - since the previous code block does not contain a plugin entry as a list item, this one shouldn't either.

Reply:
You're right, I simpy added it because at the time I forgot to add `  - ` in front of `{name: ...}` and as such got the mentioned error message while testing, and therefore added it to the `.html` file as well. I removed it now.